docs/swagger.yaml

```diff
@@ -78,6 +97,27 @@ paths:
           schema:
             type: string
             example: 2000010
+        - in: query
+          required: false
+          name: columns
```

Comment:
Do we want this to be even more specific for only the csv response? Maybe call it columnsInCSV and only use it when that response-type is requested?